html_css_dom/fetch/README.md

```diff
@@ -56,7 +56,7 @@ doSomething(result => {
 
 
 ```
-With modern functions, we attach our callbacks to the returned promises instead. Promises return a promise. Adding a `.then()` creates a promise chain:
+With modern functions, we attach our callbacks to the returned promises instead. __Promises return a promise.__ Adding a `.then()` creates a promise chain:
 
 ```js
 doSomething().then(result => {
```

Comment:
Question here is actually about the next paragraph about the code block: https://github.com/joinpursuit/Pursuit-Core-Web/pull/136/files#diff-e64d184dfc27be0878a9dd8ea59a634c1f96640e6e28e9b62a9758f3be790e23R75
```
The `.then` takes in two optional arguments. The first argument is a function that will be fired upon the promise succeeding, the second argument is the function that will be fired upon the promise failing.
```
Do we need to spell out that the callback to `then` itself takes an argument (called `result` in the code block), and that `result` is whatever the previous promise returns?

Might also be good to point out that if you are going to chain together promises with `.then`, your callbacks will all probably need to return something to the next `.then()`. The result of the original promise is not in scope of the second, third, fourth `then()`. I'm guessing this is easy to overlook and is the source of a lot of bugs.

Reply:
I think those suggestions make a lot of sense. I will update the code to include those.